src/Styles/Login/LoginPage.css

```diff
@@ -58,9 +66,10 @@
   padding: 3% 0;
   border: 2px solid var(--secondary-4);
   border-radius: 8px;
+  cursor: pointer;
 }
 .loginFormMainDiv > a {
```

Comment:
Anchor tag mentioned here is being affected by CSS? As Login form component has got no anchor tag.